### PR TITLE
Adding member names.

### PR DESCRIPTION
**Pull request by amiangshu**
Each project's readme.md should include a list of all the members full name and GitHub username.

### CODE REVIEW

**Review by davidpogosian — COMMENTED** [review]
where the rest of members??

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+We are aiming to develop a learning mangement system, that allows....
+
+#Members:
+1. David Bogosian
+2.
```
Comment (101KiddChang010) [inline README.md:5]: Jacob Rule

Comment (ReiPullum) [reply README.md:5]: replace Bogosian with Pogosian

[the remark this answers is not in the document]
Comment (amiangshu) [reply README.md]: @davidpogosian Please check again.

**Review by davidpogosian — APPROVED** [review]
very good.

**Pull request merged**